mlflow/tracing/utils/timeout.py

```diff
@@ -0,0 +1,173 @@
+import atexit
+import logging
+import threading
+import time
+from collections import OrderedDict
+
+from cachetools import Cache, TTLCache, _TimedCache
+
+from mlflow.entities.span_event import SpanEvent
+from mlflow.entities.span_status import SpanStatusCode
+from mlflow.environment_variables import (
+    MLFLOW_TRACE_BUFFER_MAX_SIZE,
+    MLFLOW_TRACE_BUFFER_TTL_SECONDS,
+    MLFLOW_TRACE_TIMEOUT_CHECK_INTERVAL_SECONDS,
+    MLFLOW_TRACE_TIMEOUT_SECONDS,
+)
+from mlflow.exceptions import MlflowTracingException
+
+_logger = logging.getLogger(__name__)
+
+_TRACE_EXPIRATION_MSG = (
+    "Trace {request_id} is automatically halted by MLflow due to timeout ({ttl} seconds). "
```

Comment:
```suggestion
    "Trace {request_id} timed out after {ttl} seconds). "
```

can we simplify the message here?